pages/friday_mac.md

```diff
@@ -4,85 +4,42 @@ title: Friday Setup - Mac
 permalink: /frisetupmac/
 ---
 
-While Macs come with Python 2.7.x pre-installed, we'll be working with Python 3 during this session so there's some light installation ahead of us. This will require administrative priveleges to your computer since the python installation is usually done to protected directories.
+While Macs come with Python 2.7.x pre-installed, we'll be working with Python 3 during this session so there's some light installation ahead of us. This will require administrative priveleges to your computer since the Python installation is usually done to protected directories.
 
 Along the way, we'll also set up a more advanced text editor that will help us by providing some really specific programming features.
 
-# Installing the Command-Line Tools
+# Installing Python 3.6 for Mac
 
-If you don't have the command line tools provided by Apple already set up, you'll need to do that.
+To install Python 3 for Mac:
 
-You can check whether you have them already by:
-
-  1. opening `Terminal` (you can find it in your `Utilities` directory inside of `Applications`)
-  2. typing in the following and then pressing `enter`:
-
-{% highlight bash %}
-xcode-select --install
-{% endhighlight %}
-
-If you get a message that looks like the following, you're already ready to go, otherwise it will begin the installation.
-
-~~~~
-xcode-select: error: command line tools are already installed, use "Software Update" to install updates
-~~~~
-
-__Backup method if the first doesn't work:__
-
-  1. Go [here](https://developer.apple.com/downloads/index.action) after logging in with any AppleID
-  2. Select and download the "Command Line Tools" for your respective version of OS X
-
-
-# Installing Homebrew
-
-[Homebrew](http://brew.sh) is a [package manager](https://en.wikipedia.org/wiki/Package_manager) for Mac. It will help us quickly install Python 3.
-
-  1. Open up the `Terminal` application (if you haven't already)
-  2. Copy and paste the following into the command window and press `enter`
-
-{% highlight bash %}
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-{% endhighlight %}
-
-During this process, you will prompted for your password (again, you'll need to have administrative priveleges). Don't worry if characters don't show up as you type; that's just how it keeps your password hidden from others' prying eyes :) !
-
-# Installing python3
-
-With Homebrew now installed, you can install `python3` with the following command in the `Terminal`.
-
-{% highlight bash %}
-brew install python3
-{% endhighlight %}
+* Download the package available here: [Python 3.6.0 Installation Package for Mac](https://www.python.org/ftp/python/3.6.0/python-3.6.0-macosx10.6.pkg)
```

Comment:
Only thing I'd maybe suggest here is to link to `https://www.python.org/downloads/` instead, so that we don't have to update this when 3.6.1 comes out.